src/image-default.yaml

```diff
@@ -27,3 +27,8 @@ squashfs-compression: zstd
 vmware-hw-version: 13
 vmware-os-type: rhel7_64Guest
 vmware-secure-boot: true
+
+# Defaults for AWS
+aws-imds-support: "v2.0"
+aws-volume-type: "gp3"
+aws-boot-mode: "uefi-preferred"
```

Comment:
this knob should probably be architecture specific. Since the only option for aarch64 is uefi I think we can get by with still only having one knob, just mention x86_64:

```
aws-boot-mode-x86-64: "uefi-preferred"
```